test/kixi/integration/base.clj

```diff
@@ -110,46 +121,48 @@
   [s]
   (client/post schema-url
                {:form-params {:schema s}
-                :content-type :transit+json
-                :accept :transit+json
-                :throw-exceptions false
-                :transit-opts {:encode t/write-handlers
-                               :decode t/read-handlers}}))
+                :content-type :json
+                :accept :json
+                :throw-exceptions false}))
+
 (defn wait-for-url
   ([url]
-   (wait-for-url url 10))
-  ([url tries]
-   (when (pos? tries)
-     (Thread/sleep 500)
+   (wait-for-url url wait-tries 1 nil))
+  ([url tries cnt last-result]
+   (if (<= cnt tries)
      (let [md (client/get url
-                          {:accept :transit+json
-                           :as :stream
-                           :throw-exceptions false
-                           :transit-opts {:encode t/write-handlers
-                                          :decode t/read-handlers}})]
+                          {:accept :json
+                           :throw-exceptions false})]
        (if (= 404 (:status md))
-         (recur url (dec tries))
-         md)))))
+         (do
+           (when (zero? (mod cnt every-count-tries-emit))
+             (prn (str "Waited " cnt " times for " url ". Getting: " (:status md))))
```

Comment:
AFAIK, println and prn both output to the same place, except `println` auto formats and adds spaces, iirc.